packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -33,24 +33,57 @@ handler.listPRs = async function listPRs(): Promise<WatchPR[]> {
   const query = datastore.createQuery(TABLE).order('created');
   const [prs] = await datastore.runQuery(query);
   const result: WatchPR[] = [];
-  for (const pr of prs) {
-    const created = new Date(pr.created).getTime();
+  for (let x = 0; x < prs.length; x += 5) {
+    if (prs[x + 1] === undefined) {
```

Comment:
I think you can get most of this same behavior with a few less lines of code by doing something like this:

```js
const WORK_SIZE = 4;
while (prs.length) {
  const work = prs.splice(0, WORK_SIZE);
}
```

:point_up:  and I would do this in the handler method `  app.on(['schedule.repository'], async context => {` rather than in the listing of work.